client/mapHandler.cpp

```diff
@@ -1210,6 +1199,9 @@ bool CMapHandler::printObject(const CGObjectInstance * obj, bool fadein)
 	const int tilesW = bitmap->width()/32;
 	const int tilesH = bitmap->height()/32;
 
+	auto ttilesWidth = ttiles.shape()[1];
+	auto ttilesHeight = ttiles.shape()[1];

```

Comment:
Same value?

Reply:
Good spot. This kinda works as long as maps are square ;)